refined/shared/src/main/scala/monocle/refined/strings.scala

```diff
@@ -1,16 +1,16 @@
 package monocle.refined
 
-import eu.timepit.refined.api.Refined
+import eu.timepit.refined.api.{Validate, Refined}
 import eu.timepit.refined.string.StartsWith
 import monocle._
 
 object strings extends StringsInstances
 
 trait StringsInstances {
 
-  def startsWith(prefix: String): Prism[String, StartsWithString[prefix.type]] = {
+  def startsWith(prefix: String)(implicit v: Validate[String, StartsWith[prefix.type]]): Prism[String, StartsWithString[prefix.type]] = {
     Prism.partial[String, Refined[String, StartsWith[prefix.type]]] {
-      case string if string.startsWith(prefix) => Refined.unsafeApply(string)
+      case string if v.isValid(prefix) => Refined.unsafeApply(string)
```

Comment:
Just noticed that this should be `v.isValid(string)` since `prefix` always satisfies `StartsWith[prefix.type]`. :-)